assembly/index.ts

```diff
@@ -40,7 +39,7 @@ function handle(input: Uint8Array): Uint8Array { // vec<u8>
       break;
     case Action.SelfEvict:
       const allowance = u128.from<u32>(0);
-      setRentAllowance(allowance)
+      setRentAllowance(allowance);
```

Comment:
There are a few good reminders to actually add a linter, which is already on my todo list. :)